### PR TITLE
chore : updated eslint and cross-spawn versions

### PR DESCRIPTION
**Pull request by khushail**
*Issue #, if available:*

- Updated with reference to this internal ticket -V1586002293
   - cross-spawn version to 7.0.3 and above 
   -  Eslint to 9.15.0
 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package.json
Comment (ashishdhingra) [inline package.json:1]: Running `npm ls cross-spawn` (after running `yarn`) shows below dependency tree:
```
stale-issue-cleanup@1.0.0 /Users/ashdhin/source/GitHub/aws-actions/stale-issue-cleanup
├─┬ @actions/github@2.2.0
│ └─┬ @octokit/rest@16.43.2
│   └─┬ universal-user-agent@4.0.1
│     └─┬ os-name@3.1.0
│       └─┬ windows-release@3.3.3
│         └─┬ execa@1.0.0
│           └── cross-spawn@6.0.5
├─┬ eslint@8.34.0
│ └── cross-spawn@7.0.3
└─┬ jest@29.7.0
  └─┬ @jest/core@29.7.0
    └─┬ jest-changed-files@29.7.0
      └─┬ execa@5.1.1
        └── cross-spawn@7.0.3
```
So, we should have updated all packages to corresponding versions which bring `cross-spawn` dependency.

**Review by ashishdhingra — CHANGES_REQUESTED** [review]

File: package.json
Comment (khushail) [reply package.json:1]: ran the commands as suggested and this shows in the updated tree-

Screenshot 2024-12-03 at 5 50 21 PM

Comment (ashishdhingra) [reply package.json:1]: Please also include `package.json` in the commit. Per output of `npm ls cross-spawn` command, we need to update the `@actions/github`, `eslint` and `jest` package versions which refer to the later `cross-spawn` version (most likely these are latest versions). So, here are the steps:
- Run `rm -rf node_modules yarn.lock`.
- Update `package.json` to bring in latest versions of the 3 packages in question (use `^` prefix while referring version)
- Run `yarn install`. This should generate new `yarn.lock`.
- Update your PR to include both `package.json` and `yarn.lock` (I would rebase and squash to one commit to keep GitHub history clean).

Comment (ashishdhingra) [reply package.json:1]: Based on our testing, we cannot update `@actions/github` since after version `2.2.0`, constructor `github.GitHub()` was removed.

Comment (khushail) [reply package.json:1]: Done

**Review by ashishdhingra — APPROVED** [review]

**Pull request closed without merging**